app/services/form_response.rb

```diff
@@ -0,0 +1,17 @@
+class FormResponse
+  def initialize(success:, errors:, extra: {})
+    @success = success
+    @errors = errors
```

Comment:
should we have this warn or throw if `errors` is an array? (so we can be consistent and use hashes only)